src/test/shared/logger/winstonToolkitLogger.test.ts

```diff
@@ -10,21 +10,22 @@ import * as fs from 'fs-extra'
 import { WinstonToolkitLogger } from '../../../shared/logger/winstonToolkitLogger'
 import { MockOutputChannel } from '../../mockOutputChannel'
 import { assertThrowsError } from '../utilities/assertUtils'
+import { waitUntil } from '../../../shared/utilities/timeoutUtils'
 
-describe('WinstonToolkitLogger', function() {
+describe('WinstonToolkitLogger', function () {
```

Comment:
This is the prettier default since 2.0 https://github.com/prettier/prettier/pull/3903

@JadenSimon is prettier formatting your commits?

Reply:
Apparently not, I'll look into it.